pkg/api/types.go

```diff
@@ -370,6 +370,9 @@ type PersistentVolumeSource struct {
 	// ScaleIO represents a ScaleIO persistent volume attached and mounted on Kubernetes nodes.
 	// +optional
 	ScaleIO *ScaleIOVolumeSource
+	// LocalStorage represents a local volume accessible only from one node
+	// +optional
+	LocalStorage *LocalStorageVolumeSource
```

Comment:
`LocalStorage` -> `Local`? Or `LocalVolume`, following `PortworxVolume` and `PortworxVolumeSource` precedent?